src/main/java/Duke.java

```diff
@@ -1,10 +1,66 @@
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(INDENTATIONS + HORIZONTAL_LINES +
+                INDENTATIONS + "Hello! I'm Duke\n" +
+                INDENTATIONS + "What can I do for you?\n" +
+                INDENTATIONS + HORIZONTAL_LINES);
+        Scanner in = new Scanner(System.in);
+        String readLine;
+        Task[] tasks = new Task[100];
+        int taskIndex = 0;
+        while(true) {
+            readLine = in.nextLine();
+            if(readLine.equals("bye")) {
+                System.out.println(INDENTATIONS + HORIZONTAL_LINES +
+                        INDENTATIONS + "Bye. Hope to see you again soon!\n" +
+                        INDENTATIONS + HORIZONTAL_LINES);
+                break;
+            } else if (readLine.equals("list")) {
+                // list out the tasks and status in todo
+                System.out.println(INDENTATIONS + HORIZONTAL_LINES +
+                        INDENTATIONS + "Here are the tasks in your list: ");
+
+                for (int i = 0; i < taskIndex; ++i) {
+                    int taskNumber = i+1;
+                    System.out.println(INDENTATIONS + taskNumber + "." +
+                            "[" + tasks[i].getStatusIcon() + "] " + tasks[i].getTaskDescription());
+                }
+
+                System.out.println(INDENTATIONS + HORIZONTAL_LINES);
+
+            } else if (readLine.startsWith("mark")) {
+                String toMark = readLine.substring(5);
+                toMark = toMark.trim();
+                int indexToMark = Integer.parseInt(toMark) - 1;
+                tasks[indexToMark].setDone(true);
+                System.out.println(INDENTATIONS + "Nice! I've marked this task as done:\n" +
+                        INDENTATIONS + "  [X] " + tasks[indexToMark].getTaskDescription() + '\n' +
+                        INDENTATIONS + HORIZONTAL_LINES);
+
+            } else if (readLine.startsWith("unmark")) {
+                String toMark = readLine.substring(7);
+                toMark = toMark.trim();
+                int indexToMark = Integer.parseInt(toMark) - 1;
+                tasks[indexToMark].setDone(false);
+                System.out.println(INDENTATIONS + "OK, I've marked this task as not done yet: \n" +
+                        INDENTATIONS + "  [ ] " + tasks[indexToMark].getTaskDescription() + '\n' +
+                        INDENTATIONS + HORIZONTAL_LINES);
+                
+            }else {
+                // add new tasks into todo
+                tasks[taskIndex] = new Task(readLine);
+                System.out.println(INDENTATIONS + HORIZONTAL_LINES +
+                        INDENTATIONS + "added: " + readLine + '\n' +
+                        INDENTATIONS + HORIZONTAL_LINES);
+                taskIndex++;
+            }
```

Comment:
A `switch`/`case` statement might be more readable than `if`/`else` in this situation in my opinion.